web_external/ImagesGallery/Detail/imageDetailsPage.jade

```diff
@@ -86,9 +86,10 @@
       tr
         td.isic-image-details-table-section-header(colspan=2) Tags
       tr
-        td
+        td(colspan=2)
           each value in image.get('notes').tags
-            span.label.label-default= value
-        td
+            //- Labels separated by a space
+            span.label.label-default.isic-image-details-tag= value
+            | &#x20;
```

Comment:
Why not use `&nbsp;`? It's a little more idiomatic for HTML than reaching into Unicode.